src/literals.py

```diff
@@ -10,3 +11,9 @@
 REL_NAME = "kafka-client"
 CHARM_USERS = ["sync"]
 TLS_RELATION = "certificates"
+
+SECURITY_PROTOCOL_PORTS = {
+    "SASL_PLAINTEXT": (9092, 19092),
+    "SASL_SSL": (9093, 19093),
+    "SSL": (9094, 19094),
+}
```

Comment:
SUGGESTION

Maybe it is really overengineered and too much, but let's see what other people think about writing this as

```
from dataclasses import dataclass
from typing import Literal, Dict

AuthMechanism = Literal["SASL_PLAINTEXT", "SASL_SSL", "SSL"]


@dataclass
class Ports:
    external: int
    internal: int


SECURITY_PROTOCOL_PORTS: Dict[AuthMechanism, Ports] = {
    "SASL_PLAINTEXT": Ports(9092, 19092),
    "SASL_SSL": Ports(9093, 19093),
    "SSL": Ports(9094, 19094),
}

```